itests/hive-unit-hadoop2/src/test/java/org/apache/hadoop/hive/metastore/security/TestHadoopAuthBridge23.java

```diff
@@ -94,16 +94,15 @@ public Server() throws TTransportException {
         super();
       }
       @Override
-      public TTransportFactory createTransportFactory(Map<String, String> saslProps)
-      throws TTransportException {
+      public TSaslServerTransport.Factory createSaslServerTransportFactory(Map<String, String> saslProps) {
         TSaslServerTransport.Factory transFactory =
           new TSaslServerTransport.Factory();
         transFactory.addServerDefinition(AuthMethod.DIGEST.getMechanismName(),
             null, SaslRpcServer.SASL_DEFAULT_REALM,
             saslProps,
             new SaslDigestCallbackHandler(secretManager));
 
-        return new TUGIAssumingTransportFactory(transFactory, realUgi);
```

Comment:
why this change is required ?

Reply:
Earlier HiveMetaStore was using createTransportFactory to create a transport factory. But now we are not using it. Instead, we are using createSaslServerTransportFactory, which should not create a wrapped transport.